qmerge.c

```diff
@@ -919,12 +919,17 @@ pkg_merge(int level, const depend_atom *atom, const struct pkg_t *pkg)
 	 * name suggests, bug #660508, usage of BINPKG_COMPRESS,
 	 * due to the minimal nature of where we run, we cannot rely on file
 	 * or GNU tar, so have to do some laymans MAGIC hunting ourselves */
-	compr = "j";  /* default: bzip2 */
+	compr = "I brotli"; /* default: brotli; has no magic header */
 	{
-		/* bz2: 2-byte: 'B' 'Z'                  at byte 0
-		 * gz:  4-byte:  1f  8b                  at byte 0
+		/* bz2: 3-byte: 'B' 'Z' 'h'              at byte 0
+		 * gz:  2-byte:  1f  8b                  at byte 0
 		 * xz:  4-byte: '7' 'z' 'X' 'Z'          at byte 1
-		 * tar: 6-byte: 'u' 's' 't' 'a' 'r' \0   at byte 257 */
+		 * tar: 6-byte: 'u' 's' 't' 'a' 'r' \0   at byte 257
+		 * lz4: 4-byte:   4  22  4d  18          at byte 0
+		 * zst: 4-byte: 22-28 b5 2f  fd          at byte 0
```

Comment:
I believe it's called zstd, if it messes up alignment, then so be it (change all lines)

Reply:
The program is called "zstd", but the extension is called "zst"